demo/simple-walkthrough/Getting Started with Dataflux Dataset for PyTorch with Google Cloud Storage.ipynb

```diff
@@ -647,6 +642,130 @@
     {
       "cell_type": "markdown",
       "source": [
+        "### Data Listing\n",
+        "\n",
+        "When working with cloud storage solutions such as GCS, data listing performance is critical because we will need it to support the map-style datasets' random access patterns.\n",
```

Comment:
"When working with cloud storage solutions such as GCS, data listing performance is critical to enabling support for map-style datasets' random access patterns.

Reply:
done.